src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -283,20 +283,44 @@
     "themes": [
         {
             "name": "light",
-            "window":{
+            "window":
+            {
                 "applicationTheme": "light"
+            },
+            "tab":
+            {
+                "background": "terminalBackground",
+                "unfocusedBackground": "#00000000"
+            },
+            "tabRow":
+            {
+                "unfocusedBackground": "#FFFFFFFF"
             }
         },
         {
             "name": "dark",
-            "window":{
+            "window":
+            {
                 "applicationTheme": "dark"
+            },
+            "tab":
+            {
+                "background": "terminalBackground",
+                "unfocusedBackground": "#00000000",
+            },
+            "tabRow":
+            {
+                "unfocusedBackground": "#333333FF"

```

Comment:
hey you get to close out this really old bug about how you can't tell when the window is inactive!

Reply:
hey I already closed that one (#4862) in #13049